### DISCUSSION TITLE
Will my own themes overwritten in the next update?

### DISCUSSION THREAD

**Discussion by glow12121** [datenstrom/community#416] [Ask a question] [answered]
If I make changes to, say, theme Berlin - will they be overwritten during the next update? Does this imply I need to create a separate theme?

Many thanks.

**Comment by schulle4u** [comment, marked as answer]
The updater takes care of modifications to extensions and won't overwrite them, unless you use the "Force update" option. Only API changes are applied, e. g. if a file's location changes.

**Comment by glow12121** [comment]
Okay, thanks!
